app/utils/sanity/start.go

```diff
@@ -78,10 +82,23 @@ var (
 		},
 	}
 
-	checkRangeItems = []rangeItem{{
-		enumRange: []int{int(state.DeliverTxsExecModeSerial), state.DeliverTxsExecModeParallel},
-		name:      state.FlagDeliverTxsExecMode,
-	},
+	checkRangeItems = []rangeItem{
+		{
+			enumRange: []int{int(state.DeliverTxsExecModeSerial), state.DeliverTxsExecModeParallel},
+			name:      state.FlagDeliverTxsExecMode,
+		},
+	}
+
+	conflictPairOpts = []conflictPairWithOption{
+		{
+			conflictPair: conflictPair{
+				configA: boolItem{name: iavl.FlagIavlEnableFastStorage, value: true},
+				configB: funcItem{name: "Upgraded to fast IAVL", expect: false, f: appstatus.IsFastStorageStrategy},
+			},
+			option: boolItem{name: iavl.FlagIavlEnableAutoUpgradeFSS, value: false,
```

Comment:
can rename checkOption, and set first location

Reply:
rename `option` to `checkOption` to the first location, and put tips at the end of conflictPairWithOption